main.go

```diff
@@ -43,9 +52,20 @@ func main() {
 	cancel := make(chan struct{})
 
 	for _, service := range services {
-		if service.Port == *ignorePort {
-			continue
-		}
+
+		var shouldSkipService = false
+
+		for _, port := range ignorePorts {
+	        if service.Port == port {
+	        	log.Printf("Ignoring Service on Port %d", port)
+	            shouldSkipService = true
+	        }
+    	}
+
+    	if shouldSkipService == true {
```

Comment:
Instead of using a helper variable for doing a `continue` on the outer loop, you could [label](https://golang.org/ref/spec#Labeled_statements) the outer loop and use the continue in the inner loop.

Even better than using an slice, you could use a map, e.g. a `map[int]struct{}` and basically use it as a set. This way, you wouldn't need a loop here and could just test whether the map has `service.Port` as key.